### PR TITLE
Add Visual Regression Testing

### PR DESCRIPTION
**Pull request by mojoaxel**
**https://github.com/mojoaxel/awesome-regression-testing**

***Regression Testing is an underestimated topic. This list tries to give an overview about the topic.***

# By submitting this pull request I confirm I've read and complied with the below requirements.

- [x] I have read and understood the [contribution guidelines](https://github.com/sindresorhus/awesome/blob/master/contributing.md) and the [instructions for creating a list](https://github.com/sindresorhus/awesome/blob/master/create-list.md).
- [x] This pull request has a descriptive title.<br>For example, `Add Name of List`, not `Update readme.md` or `Add awesome list`.
- The entry in the Awesome list should:
	- [x] Include a short description about the project/theme of the list. **It should not describe the list itself.**<br>Example: `- [Fish](…) - User-friendly shell.`, not `- [Fish](…) - Resources for Fish.`.
	- [x] Be added at the bottom of the appropriate category.
- The list I'm submitting complies with these requirements:
	- [x] **Has been around for at least 30 days.**<br>That means 30 days from either the first real commit or when it was open-sourced. Whatever is most recent.
	- [x] It's the result of hard work and the best I could possibly produce.
	- [x] Non-generated Markdown file in a GitHub repo.
	- [x] **Includes a succinct description of the project/theme at the top of the readme.** [(Example)](https://github.com/willempienaar/awesome-quantified-self)
	- [x] The repo should have `awesome-list` & `awesome` as [GitHub topics](https://help.github.com/articles/about-topics). I encourage you to add more relevant topics.
	- [x] Not a duplicate.
	- [x] Only has awesome items. Awesome lists are curations of the best, not everything.
	- [ ] Includes a project logo/illustration whenever possible.
		- Placed at the top-right of the readme. [(Example)](https://github.com/sindresorhus/awesome-electron)
		- The image should link to the project website or any relevant website.
		- The image should be high-DPI. Set it to maximum half the width of the original image.
	- [x] Entries have a description, unless the title is descriptive enough by itself. It rarely is though.
	- [x] Has the [Awesome badge](https://github.com/sindresorhus/awesome/blob/master/awesome.md#awesome-badge) on the right side of the list heading,
	- [x] Has a Table of Contents section.
		- [x] Should be named `Contents`, not `Table of Contents`.
		- [x] Should be the first section in the list.
		- [x] Should only have one level of sub-lists, preferably none.
	- [x] Has an [appropriate license](https://github.com/sindresorhus/awesome/blob/master/awesome.md#choose-an-appropriate-license).
		- [x] That means something like CC0, **not a code licence like MIT, BSD, Apache, etc.**
		- [x] If you use a license badge, it should be SVG, not PNG.
	- [x] Has [contribution guidelines](https://github.com/sindresorhus/awesome/blob/master/awesome.md#include-contribution-guidelines).
		- [x] The file should be named `contributing.md`. Casing is up to you.
	- [x] Has consistent formatting and proper spelling/grammar.
		- [x] Each link description starts with an uppercase character and ends with a period.<br>Example: `- [AVA](…) - JavaScript test runner.`
		- [x] Drop all the `A` / `An` prefixes in the descriptions.
		- [x] Consistent and correct naming. For example, `Node.js`, not `NodeJS` or `node.js`.
	- [x] Doesn't include a Travis badge.<br>You can still use Travis for list linting, but the badge has no value in the readme.
- [ ] Go to the top and read it again.

### CODE REVIEW

**Comment by sindresorhus** [comment]
If you need to have "visual" in parens, I think that's a good sign you should rename the list to "Visual Regression Testing".

**Comment by sindresorhus** [comment]
>  Include a short description about the project/theme of the list. It should not describe the list itself.
Example: - [Fish](…) - User-friendly shell., not - [Fish](…) - Resources for Fish..

It should be about the topic, not just a repeat of the title.

---

>  Includes a succinct description of the project/theme at the top of the readme. (Example)

Not done.

---

>  Drop all the A / An prefixes in the descriptions.

Not done.

---

> DalekJS - Not maintained any longer.

Don't include unmaintained stuff. It's supposed to be an **awesome** list.

---

All the items in the `Online services` section needs descriptions.

**Comment by mojoaxel** [comment]
@sindresorhus OK, Done!

**Comment by sindresorhus** [comment]
> If you need to have "visual" in parens, I think that's a good sign you should rename the list to "Visual Regression Testing".

You need to update the title the readme addition here and the PR title.

File: readme.md [hunk 1]
```diff
@@ -553,6 +560,7 @@
 - [Homematic](https://github.com/hobbyquaker/awesome-homematic) - Smart home devices.
 - [Ledger](https://github.com/sfischer13/awesome-ledger) - Double-entry accounting on the command-line.
 - [Uncopyright](https://github.com/johnjago/awesome-uncopyright) - Public domain works.
+- [Regression Testing](https://github.com/mojoaxel/awesome-regression-testing) - Automated (Visual) Regression Testing.
```
Comment (sindresorhus) [inline readme.md]: > The entry in the Awesome list should:
 Include a short description about the project/theme of the list. **It should not describe the list itself.**
Example: - [Fish](…) - User-friendly shell., not - [Fish](…) - Resources for Fish..

**Comment by sindresorhus** [comment]
> General information on regression testing

Too long. "Regression testing" is implied.

> Slideshows, Talks and Videos

Inconsistent casing with the rest.

> Misc

Use the full word.

> BackTrac - Cloud based visual regression tool. Uses headless firefox and proprietary comparison algorithm (detect layout shifts)

Missing a dot at the end here.

> Slideshows, Talks and Videos

The items in this section needs descriptions. Could describe what the different tools are; Wraith, Shoov, PhantomCSS, ..

> See the Awesome Regression Testing Contribution Guide

Too long. "Contribution guide" is enough.

> helpfull

Typo

**Comment by sindresorhus** [comment]
@mojoaxel ping

**Comment by mojoaxel** [comment]
@sindresorhus pong

**Comment by sindresorhus** [comment]
> Slideshows, Talks and Videos
Inconsistent casing with the rest.

You also need to update the "Contents" entry.

**Comment by mojoaxel** [comment]
> You also need to update the "Contents" entry.

Wow, you really spot every little detail! 🏅 
Done...

**Comment by sindresorhus** [comment]
Great. Thanks for your patience.

**Comment by sindresorhus** [comment]
> Wow, you really spot every little detail! 🏅

Awesome lists should be **awesome** in every sense of the word.

**Comment by sindresorhus** [comment]
I will tweet the list later this week from this account: https://twitter.com/awesome__re

**Pull request merged**